CONTRIBUTING.adoc

```diff
@@ -50,101 +45,98 @@ https://github.com/openshift/origin[origin repo].
-    `/etc/sysconfig/docker` and restart the Docker daemon. For more details on
-    controlling Docker options with systemd, please refer https://docs.docker.com/engine/admin/systemd/[here].
-    Otherwise, add "--insecure-registry 172.30.0.0/16" to the Docker daemon
-    invocation, for example:
+Follow the installation steps to install https://git-for-windows.github.io/[`git` for Windows]
+and https://docs.docker.com/docker-for-windows/install/[`docker`].
+You will need to build `linux/amd64` binaries for the OpenShift server; if you
+want to do the builds locally, you will need to follow the instructions to
+install https://golang.org/doc/install#windows[the `go` programming language].
 
-        $ docker daemon --insecure-registry 172.30.0.0/16
+===== Fedora
 
-7.  Then, the OpenShift firewalld rules are also a work in progress. For now it
-    is easiest to disable firewalld altogether:
-
-        $ sudo systemctl stop firewalld
-
-8.  Firewalld will start again on your next reboot, but you can manually restart
-    it with this command when you are done running OpenShift:
+Install `go`, `git` and `docker` with:+
+----
+$ sudo dnf install golang git docker-latest
+----
 
-        $ sudo systemctl start firewalld
+===== CentOS / RHEL
 
-9.  Now change into the directory with the OpenShift binaries, and start the
-    OpenShift server:
+Install `go`, `git` and `docker` with:+
+----
+$ sudo yum install golang git docker
+----
 
-        $ cd _output/local/bin/linux/amd64
-        $ sudo ./openshift start
+==== Configuring a Development Environment
 
+1. Create a Go workspace directory: +
 +
-NOTE: Replace "linux/amd64" with the appropriate value for your platform/architecture.
-
-10.  Launch another terminal, change into the same directory you started
-     OpenShift, and deploy the private docker registry within OpenShift
-     with the following commands:
-
-        $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
-        $ ./oadm registry -n default --config=openshift.local.config/master/admin.kubeconfig
-
-11.  If it is not there already, add the current directory to the $PATH, so you
-     can leverage the OpenShift commands elsewhere.
-
-12.  You are now ready to edit the source, rebuild and restart OpenShift to
-     test your changes.
-
-13.  NOTE:  to properly stop OpenShift and clean up, so that you can start
-     fresh instance of OpenShift, execute:
-
-        $ sudo pkill -x openshift
-        $ docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
-        $ mount | grep "openshift.local.volumes" | awk '{ print $3}' | xargs -l -r sudo umount
-        $ cd <to the dir you ran openshift start> ; sudo rm -rf openshift.local.*
+----
+$ mkdir $HOME/go
+----
+2. In your `.bashrc` file or `.bash_profile` file, set a GOPATH and update your PATH: +
++
+----
+export GOPATH=$HOME/go
+export PATH=$PATH:$GOPATH/bin
+export OS_OUTPUT_GOPATH=1
```

Comment:
These environment variables are the basics of a normal Golang development environment -- you may not _need_ them to run specific `make` targets but you need a `$GOPATH` to use normal Go tools.